.github/workflows/shisenapp_ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ShisenApp CI
+on:
+  push:
+    branches: 
+      - master 
+  pull_request:
+    branches: 
+      - master 
+
+jobs:
+  build-test-deploy:
+    name: Build, test, and deploy the project
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build and test
+        run: |
+          npm install
+          npm run build
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@4.1.0
+        with:
+          branch: gh-pages
+          folder: dist 
```

Comment:
Replace this with `build` as the output directory of the build process is `build` instead of `dist`. As seen in [the last asction result](https://github.com/ichiro-its/shisen-app/runs/2078254559).